src/Service/OriginKeyService.php

```diff
@@ -74,17 +81,17 @@ public function getOriginKeyForOrigin(string $host): OriginKeyModel
         try {
             $response = $this->adyenCheckoutUtilityService->originKeys($params);
         } catch (AdyenException $e) {
-            die($e->getMessage());
-            //TODO log error
+            $this->loggerService->error($e->getMessage());
+            exit;
```

Comment:
what happens if it exits? Blank page or Shopware can handle it?

Reply:
The response is empty, since the originKey service is not used yet the empty response is not handled